guidelines/terms/20/blocks-of-text.html

```diff
@@ -1,6 +1,6 @@
-<dt><dfn id="dfn-blocks-of-text">blocks of text</dfn></dt>
+<dt><dfn id="dfn-blocks-of-text" data-lt="block of text">blocks of text</dfn></dt>
 <dd>
    
-   <p>more than one sentence of text</p>
+  <p>more than one sentence of <a>text</a></p>
```

Comment:
I've reported this before. This is a mistake. WCAG's "text" definition refers to programmatic text, whereas the dictionary meaning of "text" also covers images of text. Connecting "blocks of text" to WCAG's definition of text means blocks of text no longer cover images of text. That then means 1.4.8, which is the only SC using blocks of text, now no longer applies to images of text. That seems like a mistake to me, and a significant normative change.

There are still plenty of non-programmatic texts on the web. Anything rendered on canvas is effectively an image of text, and that's also the exact scenario where reflow can't be taken for granted. This is not a good change IMO.

Reply:
@WilcoFiers prior feedback from you and otherson these changes was reviewed and incorporated by the TF. For instance, the link to the definition of text has been removed from both versions of Images of Text, for the reason you cite.

If you examine the criteria specified in 1.4.8, including the four "requirements" specified in the techniques, this AAA criterion is clearly targetted at text that can be manipulated, including:
- text and background colors altered
- width is restricted to a certain number of characters
- the ability to increase text size without scrolling horizontally

And of course there is another criterion which effectively rules out the use of images of text at AAA. Given these two considerations, amongst others, it was felt appropriate and clarifying to include the link to the definition in 1.48, just as it was considered clarifying to remove it in Images of Text (both 1.4.5 and 1.4.9).

Reply:
> If you examine the criteria specified in 1.4.8, including the four "requirements" specified in the techniques, this AAA criterion is clearly targetted at text that can be manipulated via a mechansim

The point of the criteria is specifically to forbid techniques that would break the ability to manipulate text. Techniques that prevent the text from being programmatically manipulable are *exactly what this criterion needs to be able to target*. I agree with Wilco that we should omit the link from this specific case.

Reply:
```suggestion
  <p>more than one sentence of text</p>
```